src/main/java/duke/utilities/TaskList.java

```diff
@@ -0,0 +1,47 @@
+package duke.utilities; /**
+ * a class to store, delete and add new tasks
```

Comment:
Class javadoc comments should appear after imports right before the class is defined.